_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,227 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks that a role has at least one of its required owned elements.
+
+success_criterion:
+- 1.3.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role).
+
+### Expectation
+
+For each test target at least one instance of one [required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria) is present as a [semantic role](#semantic-role) for an element [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by the test target.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+When a required owned element is not an immediate child, but rather a nested descendant, certain AT have some issues recognize child in that configuration which can be negated by defining a valid role for intermediate elements.
+
+## Background
+
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+- [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
+- [HTML in ARIA](https://www.w3.org/TR/html-aria/)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Element `ul` with role `list` has atleast one child node `span` with the required owned element `listitem` as an explicit semantic role.
+
+```html
+<ul role='list'>
+	<span role='listitem'>
+	</span>
+	<span>
+	</span>
+</ul>
+```
+
+#### Passed example 2
+
+Element `ol` with role `tablist` has child node `li` with the required owned element `tab` as an explicit semantic role.
+
+```html
+<ol role='tablist'>
+	<li role='tab'>
+	</li>
+</ol>
+```
+
+#### Passed example 3
+
+Element with role `list` has `li` element which has an implicit semantic role of ´listitem´, which is a required owned element.
+
+```html
+<ul role='list'>
+	<li></li> <!-- implicit role -->
+</ul>
+```
+
+#### Passed example 4
+
+Multiple roles that each have an instance of their required owned elements present as child nodes.
+
+```html
+<table role='grid'>
+	<tr role='row'>
+		<span role='cell'>
+		</span>
+	</tr>
+</table>
+```
+
+#### Passed example 5
+
+Multiple types of required owned elements are present.
+
+```html
+<ul role='menu'>
+	<li></li> <!-- implicit role -->
+	<li role='menuitem'></li>
+	<li role='menuitemradio'></li>
```

Comment:
Do we have to write only valid examples ?
Here for instance, [`role='menuitemradio'` require the `aria-checked` property](https://www.w3.org/TR/wai-aria-1.1/#menuitemradio). I know that this test doesnt evaluate the presence of required attributes and, at this time "we don't care" about that property, but while writting snippet (and passed examples), don't we have to write valid code ? or it is assume somewhere (like a disclaimer), that the snippets we write just present the elements needed to accomplish the test, and no other ?

Reply:
Okay I open an issue#253  to make it a more general discussion. Is that ok ?